lib/Conversion/LowerDIP/LowerDIPPass.cpp

```diff
@@ -44,6 +48,23 @@ using namespace mlir::arith;
 
 namespace {
 
+Value insertZeroConstantOp(MLIRContext *ctx, OpBuilder &builder, Location loc, Type elemTy) {
```

Comment:
I think this function can be suitably placed inside `Utils.h`

Reply:
Moved.